lib/index.js

```diff
@@ -49,7 +49,7 @@
       return tag
     }
 
-    return str.replace(/\[(\d+;)?(\d+)*m/g, function(match, b1, b2) {
+    return str.replace(/\[(\d{1,3};)?(\d{1,3})?m/g, function(match, b1, b2) {
```

Comment:
How did you come with a pattern of 1 to 3 length restriction?
Are you able to provide user acceptance testing to validate that there will be no breaking change?